crates/cxx-qt-gen/src/generator/rust/fragment.rs

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use proc_macro2::TokenStream;
+use syn::{Item, Result};
+
+/// A generic Rust CXX bridge definition and source pairing
+pub struct RustFragmentPair {
+    pub cxx: Vec<TokenStream>,
+    pub source: Vec<TokenStream>,
+}
```

Comment:
```suggestion
/// A generic Rust CXX bridge definition and the corresponding implementation
pub struct RustFragmentPair {
    pub cxx_bridge: Vec<TokenStream>,
    pub implementation: Vec<TokenStream>,
}
```

I did not understand this on first read and had to see some example usages to understand what these fields indicated.

Reply:
Done

Reply:
I still think `source` is too vague for the member name and would prefer renaming it to `implementation`. Whether you want to change `cxx` to `cxx_bridge`, I don't really care.

Reply:
Oops missed that in the diff, changing....

Reply:
Done